src/main/java/com/exceptionless/exceptionlessclient/plugins/preconfigured/DuplicateCheckerPlugin.java

```diff
@@ -119,13 +111,14 @@ private void addNewHashIfPossible(long hash, long now) {
     hashes.add(TimeStampedHash.builder().hash(hash).timestamp(now).build());
   }
 
-  private long getHashCode(InnerError error) {
-    long hash = 0L;
-    while (error != null) {
-      hash += Objects.hash(error.getMessage(), error.getStackTrace());
-      error = error.getInner();
-    }
-    return hash;
+  private long getHash(Event event) {
+    return Objects.hash(
```

Comment:
I have not included the event reference id and event count as part of hash check

Reply:
You may want to take a look at this: https://github.com/exceptionless/Exceptionless.Net/blob/master/src/Exceptionless/Models/Client/Event.cs#L61-L88

Reply:
👍 Added `geo` and `value` to the list